UPGRADE.md

```diff
@@ -272,6 +272,9 @@ been removed in Contao 4.0.
 
 The `$_SESSION['TL_USER_LOGGED_IN']` flag has been removed.
 
+### $_SESSION['TL_LANGUAGE']
+
+The `$_SESSION['TL_LANGUAGE']` flag has been removed.
```

Comment:
You don't have to add this here. The variable has been deprecated since Contao 4.0.

https://github.com/contao/core-bundle/blob/4.4/DEPRECATED.md#globalstl_language-and-_sessiontl_language